### ISSUE TITLE
Cannot open changesets with only deleted features in JOSM 

### ISSUE THREAD

**Issue by planemad** [willemarcel/osmcha-django#69]
For changesets with only a deleted feature https://osmcha.mapbox.com/46918357/ `open in JOSM` loads a blank view of the world.

This should instead load the data in the changeset bbox.

**Comment by planemad** [comment]
tracking upstream https://github.com/willemarcel/osmcha-django/issues/69

**Comment by willemarcel** [comment]
Good catch, @planemad! We need to change the `import` command on the JOSM url to a `load_and_zoom` on changesets with only deleted features. I'll do it soon... Thank you!
